### ISSUE TITLE
Clone to Other View not working

### ISSUE THREAD

**Issue by raphaelmsx** [notepadqq/notepadqq#711]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/notepadqq/notepadqq/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
When right-clicking open file tab and selecting "clone to other view" nothing happens.

**Expected behavior**
Right-clicking any open file tab and selecting "clone to other view" should do exactly what is mentioned. (in other words: split screen).

**Minimal reproduction of the problem with instructions**
01: Right-click any open file tab.
02: Select "clone to other view".

**Please tell us about your environment:**
Notepadqq 1.4.2
Fedora 28
Kernel 4.16.14-300.fc28.x86_64
KDE 5.12.5
Qt 5.10.1

**Comment by JuBan1** [comment]
You're right. This isn't a bug per se, but an unimplemented feature. For now I've hidden the 'Clone to Other View' action (#714) but hopefully we actually get it implemented in the future. Right now there are other priorities.

**Comment by 9527KGG** [comment]
+1. It's really useful.

**Comment by deathtrip** [comment]
any news on split view?
it's an essential feature imo

**Comment by schriscoli** [comment]
Similar to above - running 1.4.8 on ubuntu 20.04.  Clone to other view not hidden as above and doesn't work.
